include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -47,6 +47,9 @@ static const char* const kOrtSessionOptionsConfigSetDenormalAsZero = "session.se
 // Its default value is "1"
 static const char* const kOrtSessionOptionsEnableQuantQDQ = "session.enable_quant_qdq";
 
+// Enable or disable gelu approximation in graph optimization. "0": disable; "1": enable. The default is "0".
```

Comment:
Would be good to capture in the comment here why this config param would be used, including which sort of models it's applicable to, and why it's disabled by default.

Reply:
We can add some descriptions of all configuration keys in API document later since user might not look at code here.